documentation/source/spelling_wordlist.txt

```diff
@@ -0,0 +1,87 @@
+AMS
+Aldec
+Altera
+Async
+Avalon
+AXI
+Cosimulation
+FLI
+GPI
+Gigabit
+GitHub
+HDL
+Hexdump
+IPython
+Indices
+JUnit
+Lite
+Microsemi
+MinGW
+ModelSim
+Msys
+NVC
+Questa
+Quickstart
+README
+Roadmap
```

Comment:
That's not a normal word? Or is it just the capitalization?

Reply:
It's going to differ depending on what spell checker backend enchant is configured to use, and what dictionary it has available, but on my fairly vintage system, it's indeed considered a typo (with "roadkill" being one of the suggestions :)